sycl/test/extensions/properties/properties_kernel_device_has_warning.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: %clangxx -fsycl-device-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s
```

Comment:
Can you add `-Xclang -no-opaque-pointers` to the first RUN: lines as well?

Reply:
Absolutely! They have been added.